client/src/pages/ProfileSettings/ProfileSettings.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+import useStyles from './useStyles';
+import CircularProgress from '@mui/material/CircularProgress';
+import Grid from '@mui/material/Grid';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import CssBaseline from '@mui/material/CssBaseline';
+import TabPanel from '../../components/TabPanel/TabPanel';
+import EditProfile from '../../components/EditProfile/EditProfile';
+import ProfilePhoto from '../../components/ProfilePhoto/ProfilePhoto';
+
+export default function ProfileSettings(): JSX.Element {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+  const history = useHistory();
+
+  const [value, setValue] = useState(0);
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    // loading for a split seconds until history.push works
+    return <CircularProgress />;
+  }
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+  function a11yProps(index: number) {
+    return {
+      id: `vertical-tab-${index}`,
+      'aria-controls': `vertical-tabpanel-${index}`,
+    };
+  }
+  return (
+    <Grid container className={classes.root}>
+      <CssBaseline />
+      <Grid item xs={10} sm={4}>
+        <Tabs
+          orientation="vertical"
+          value={value}
+          onChange={handleChange}
+          aria-label="Vertical tabs for setting selection"
+        >
+          <Tab label="Edit profile" {...a11yProps(0)} />
+          <Tab label="Profile Photo" {...a11yProps(1)} />
+          <Tab label="Payment" {...a11yProps(2)} />
+          <Tab label="Security" {...a11yProps(3)} />
+          <Tab label="Settings" {...a11yProps(4)} />
```

Comment:
I think actually having routes instead of tabs would be better for the user. Let's say we wanted to bring in a help center and provide direct links to where the user needs to go, this wouldn't be possible with the current structure.

Reply:
Make sense, I'll try to implement this feature using react routes and if get time will commit new changes and merge it. 🦜